### ISSUE TITLE
Add a feature and encryption method comparison table

### ISSUE THREAD

**Issue by Mygod** [shadowsocks/shadowsocks-org#22]
See also: https://github.com/shadowsocks/shadowsocks/wiki/Feature-Comparison-across-Different-Versions

**Comment by wongsyrone** [comment]
Already have one in https://github.com/shadowsocks/luci-app-shadowsocks/wiki/Encrypt-method.

**Comment by madeye** [comment]
@wongsyrone A pull request for the website?

**Comment by Mygod** [comment]
@wongsyrone Thanks but what I meant is having a table for different ports of shadowsocks.

**Comment by wongsyrone** [comment]
@Mygod  You said feature comparison, but what aspects should be considered?
@madeye  I really don't know how to modify it.

**Comment by Mygod** [comment]
See that table I mentioned. It should include TCP Fast Open, etc.

**Comment by Mygod** [comment]
That's a table, not a feature comparison table though.

**Comment by henrypijames** [comment]
I think using the Wiki page is a good idea, since it allows the developers of different SS implementations to update the features of their own software. The `shadowsocks.org` website would then have a link to the Wiki page, similar to what's proposed in #51.

**Comment by riobard** [comment]
We should probably define a standard set of features first, and allow each implementation to define their extensions.

**Comment by riobard** [comment]
I've added a page to list current implementations under active maintenance with feature comparison here https://github.com/shadowsocks/shadowsocks-org/wiki/Implementations

Please leave a comment for any missing features.

**Comment by breakwa11** [comment]
what is "Redirect mode"

**Comment by ccsexyz** [comment]
Netfilter TCP redirect

**Comment by riobard** [comment]
We should have a brief explanation of each feature, or at least links to its spec. Anyone knows what's the spec for multiuser and management API? I just copied that from the original wiki.

**Comment by riobard** [comment]
Also, should we have a section for 3rd-party implementations and clients?

**Comment by hellofwy** [comment]
From the original Wiki of Shadowsocks in python, maybe outdated:

https://github.com/shadowsocks/shadowsocks/wiki/Configure-Multiple-Users

https://github.com/shadowsocks/shadowsocks/wiki/Manage-Multiple-Users

**Comment by riobard** [comment]
@hellofwy Nice!

**Comment by Mygod** [comment]
I think it's better to combine these two tables and add "Server mode" row.

**Comment by riobard** [comment]
@Mygod I was thinking about the same until I realize the features for clients are very different from those of servers.

**Comment by Mygod** [comment]
Hmm... On second thought, we should add shadowsocks-libev (etc.) and features like TFO, UDP relay to clients table then.

**Comment by Mygod** [comment]
Also redirect/tunnel mode should be moved to clients.

**Comment by riobard** [comment]
Yeah, it's a mess right now. Haven't found a better way to organize so I just followed the format of the original wiki page.
